src/rotten-tomatoes/SpoilTomatoes.js

```diff
@@ -55,7 +54,8 @@ class SpoilTomatoes {
                 let badTomatoe = arrOfBadTomatoes[i];
                 if (!visited[badTomatoe]) {
                     visited[badTomatoe] = true;
-                    const adjacentTomatoes = this.getAdjacents(badTomatoe);
+                    let [row,] = badTomatoe;
+                    const adjacentTomatoes = this.getAdjacents(badTomatoe, this.grid[row].length - 1, this.grid.length - 1);
```

Comment:
You can update the rowSize and columnSize in the constructor and just use it everywhere.